src/javascript/static/pages/get_started.js

```diff
@@ -22,7 +25,20 @@ module.exports = {
         onUnload: () => { MenuSelector.clean(); },
     },
     BinaryOptionsForMT5: {
-        onLoad  : () => { MenuSelector.init(['what-are-binary-options', 'how-to-trade-binary', 'types-of-trades']); },
+        onLoad: () => {
+            let menu_sections = [
+                'what-are-binary-options',
+                'how-to-trade-binary',
+                'types-of-trades',
+            ];
+            BinarySocket.wait('authorize', 'website_status', 'landing_company').then(() => {
+                if (isEuCountry()) {
```

Comment:
do you need this check? think it's redundant with the usage of `data-show='-eucountry'` 🤔

Reply:
Yes, it still need to check for the `next` and `previous` on the bottom to remove it, otherwise when user press next, they will go to the `how-to-binary-mt5` page.